### PR TITLE
Chore(node feat)

### PR DESCRIPTION
**Pull request by AugustinMauroy**
# Related issue

#5308

### CODE REVIEW

File: components/Home/NodeFeat/index.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+const NodeFeature = ({ icon, heading, description }: NodeFeatureProps) => (
+  <div className={styles.text}>
+    {icon}
+    <h2 className="t-subheading2">{heading}</h2>
+    <p>{description}</p>
+  </div>
+);
+
+const NodeFeatures = () => {
+  return (
```
Comment (ovflowd) [inline components/Home/NodeFeat/index.tsx]: No need for return statement here :)

File: components/Home/NodeFeat/index.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+const NodeFeature = ({ icon, heading, description }: NodeFeatureProps) => (
+  <div className={styles.text}>
+    {icon}
+    <h2 className="t-subheading2">{heading}</h2>
+    <p>{description}</p>
+  </div>
+);
+
+const NodeFeatures = () => {
```
Comment (ovflowd) [inline components/Home/NodeFeat/index.tsx]:
```suggestion
const NodeFeatures: React.FC = () => {
```

File: components/Home/NodeFeat/index.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    description: 'pages.index.features.openSource.description',
+  },
+  {
+    icon: styled(<IoMdRocket />),
+    heading: 'pages.index.features.everywhere.title',
+    description: 'pages.index.features.everywhere.description',
+  },
+];
+
+const NodeFeature = ({ icon, heading, description }: NodeFeatureProps) => (
```
Comment (ovflowd) [inline components/Home/NodeFeat/index.tsx]:
```suggestion
const NodeFeature: React.FC<NodeFeatureProps> = ({ icon, heading, description }) => (
```

File: components/Home/NodeFeat/index.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import { cloneElement } from 'react';
+import { IoLogoNodejs, IoMdGitPullRequest, IoMdRocket } from 'react-icons/io';
+import { FormattedMessage } from 'react-intl';
+import styles from './index.module.scss';
+import type { ReactElement } from 'react';
+
+type NodeFeatureProps = {
```
Comment (ovflowd) [inline components/Home/NodeFeat/index.tsx]: The prop should be close to the component

File: styles/typography.scss
Comment (ovflowd) [inline styles/typography.scss:1]: Nice addition! But typography should be added before `./base` afaics

**Review by ovflowd — COMMENTED** [review]
So far so good! Left a few comments.

Another note... As I mentioned I think it's better if the path is `components/Home/NodeFeature`

File: components/Home/NodeFeat/index.tsx (hunk 3 above)
Comment (ovflowd) [reply components/Home/NodeFeat/index.tsx]: We can also split these two components into different files...

We can do something like:

```
- components/NodeFeatures/
  - index.tsx <-- NodeFeatures component here
  - NodeFeature/
    - index.tsx <-- NodeFeature component here
```

Comment (AugustinMauroy) [reply components/Home/NodeFeat/index.tsx]: Yeah we can split. But we will not unit test nodeFeature. So I think it's not useful.

File: styles/typography.scss
Comment (AugustinMauroy) [reply styles/typography.scss:1]: What do you think about putting them in alphabetical order?

Comment (ovflowd) [reply styles/typography.scss:1]: That's not how they should be ordered. They should be ordered by "import order" (priority)

File: components/Home/NodeFeat/index.tsx (hunk 3 above)
Comment (ovflowd) [reply components/Home/NodeFeat/index.tsx]: We don't need to unit test every component, if there's nothing to be tested ;)

The split makes sense, as a NodeFeature could be used elsewhere, + I feel that in the future we will add NodeFeatures directly to the Home Layout

**Review by mikeesto — COMMENTED** [review]
Nice one. Couple of lint errors that should be simple to fix: https://github.com/nodejs/nodejs.org/actions/runs/4834719906/jobs/8620977658?pr=5338

File: components/Home/NodeFeatures/index.test.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { render } from '@testing-library/react';
```
Comment (ovflowd) [inline components/Home/NodeFeatures/index.test.tsx]: tests should be inside `__tests__` as all other component do that

Comment (AugustinMauroy) [reply components/Home/NodeFeatures/index.test.tsx]: Yes! Wait what I have done ...

**Review by ovflowd — APPROVED** [review]
LGTM! Looks really good! Thank you for the contribution, Augustin!

cc @nodejs/website for more reviews :)

**Review by mikeesto — APPROVED** [review]

**Pull request merged**